components/Avatar/avatar.css

```diff
@@ -0,0 +1,58 @@
+@import url("../../Stylesheet/variable.css");
+body {
+  background-color: var(--color-white);
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 2rem;
+  padding: 2rem;
+}
+.content {
+  display: flex;
```

Comment:
container, content, leftNav .... use similar properties of flex. can optimize it

Reply:
have removed the flex property from leftNav.

Reply:
don't leave unused code as commented, remove it.